package.json

```diff
@@ -21,7 +21,7 @@
     "prepublish": "npm run build",
     "lint": "eslint src test",
     "unit-test": "mocha --compilers js:babel-register",
-    "test": "npm run lint --silent && npm run unit-test --silent"
+    "test": "npm run build && npm run lint --silent && npm run unit-test --silent"
```

Comment:
added this for convenience. open to reverting this change if it's not desirable.